packages/cyberstorm/src/components/Layout/Developers/ManifestValidator/ManifestValidatorLayout.tsx

```diff
@@ -37,4 +159,14 @@ export function ManifestValidatorLayout() {
   );
 }
 
+const selectOptions = [
```

Comment:
FYI `Dapper.getCurrentUser` is merged and it contains user's team info too, so at least Dapper progress is no longer a reason to use dummy data here.

Reply:
Added the usage of that 👍